src/daos/AuthorDAO.java

```diff
@@ -0,0 +1,121 @@
+
+package daos;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import beans.Author;
+import configurations.HibernateUtils;
+
+public class AuthorDAO {
+
+	public static boolean delete(String id) throws Exception {
+
```

Comment:
Please explain why we have to use try catch and throws for the same kind of exception

Reply:
Yeah I deleted unnecessary Exception caught. Please check them for me